project/Settings.scala

```diff
@@ -68,6 +68,7 @@ object Settings {
     assembly / assemblyJarName := name.value + "-" + version.value + ".jar",
     assembly / test := {},
     assembly / assemblyMergeStrategy := customMergeStrategy.value,
+    assembly / assemblyCacheOutput := false
```

Comment:
this is to speed up the build process - see https://stackoverflow.com/questions/19543530/sbt-assembly-task-runs-slowly-after-adding-some-dependencies